dist/src/main/java/io/camunda/zeebe/gateway/AtomixComponent.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.gateway;
+
+import io.atomix.cluster.AtomixCluster;
+import io.atomix.cluster.AtomixClusterBuilder;
+import io.atomix.cluster.discovery.BootstrapDiscoveryProvider;
+import io.atomix.cluster.protocol.GroupMembershipProtocol;
+import io.atomix.cluster.protocol.SwimMembershipProtocol;
+import io.atomix.utils.net.Address;
+import io.camunda.zeebe.gateway.impl.configuration.ClusterCfg;
+import io.camunda.zeebe.gateway.impl.configuration.GatewayCfg;
+import io.camunda.zeebe.gateway.impl.configuration.MembershipCfg;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ScopedProxyMode;
+import org.springframework.stereotype.Component;
+import org.springframework.web.context.annotation.ApplicationScope;
+
+@Component
+final class AtomixComponent {
+  private final GatewayCfg config;
+
+  @Autowired
+  AtomixComponent(final GatewayCfg config) {
+    this.config = config;
+  }
+
+  @Bean("atomixCluster")
```

Comment:
The `StandaloneBroker` and the `StandaloneGateway` both define which packages they scan for beans/components/configuration. Since the `AtomixComponent` is defined in the `gateway` package, it's not scanned by the `StandaloneBroker`.